spec/03-plugins/17-jwt/01-jwt_parser_spec.lua

```diff
@@ -29,6 +29,53 @@ describe("Plugin: jwt (parser)", function()
         ]], true), token)
       end
     end)
+    it("should properly encode using HS384", function()
+      local token = jwt_parser.encode({
+        name  = "John Doe",
+        admin = true,
+        sub   = "1234567890"
+      }, "secret", 'HS384')
+
+      if helpers.openresty_ver_num < 11123 then
+        assert.equal(u([[
+          eyJhbGciOiJIUzM4NCIsInR5cCI6IkpXVCJ9.eyJhZG1pbiI6d
+          HJ1ZSwibmFtZSI6IkpvaG4gRG9lIiwic3ViIjoiMTIzNDU2Nzg
+          5MCJ9.fAk3ps-s7_vQk6XpiVq5GiNjZ__cMW37kE9MEQoR6MwX
+          ELTFtSIoNOpmhOe_wnCS
+        ]], true), token)
+        assert.equal('not', 'implemented')
```

Comment:
If the feature is supposedly not implemented on some older dependency version (although I am not sure why it wouldn't), then isn't the assertion doomed to fail?

Also, 'not' and 'implemented' clearly aren't equal, so the second assertion doesn't really make any sense? If anything, `jwt_parser.encode()` will [throw an error](https://github.com/kepkin/kong/blob/51b1ca0ec9ec537ef86730cc63c20084c49751d7/kong/plugins/jwt/jwt_parser.lua#L182), not return one.

This while branch goes away anyway since we only support OpenResty 1.13.6.2 and above from now on :)